### PR TITLE
Fix grammar in design doc

### PR DESCRIPTION
**Pull request by gabrielhartmann**

### CODE REVIEW

File: docs/design.md [hunk 1]
```diff
@@ -79,25 +79,25 @@ Rackspace|Rackspace API|not supported|TBD|
...
+Healthcheck is an optional component that allows you to control the damage inflicted against your service. If enabled, Atropos kicks off the healthcheck logic defined in the Fate Book before inducing a failure. The Healthcheck component needs to return success in order for the failure run. Otherwise the Scheduler skips the current failure cycle. This ensures that we are not aggravating any existing issues and lets the cluster fire self-healing routines and recover. If a healthcheck is not defined, failures will be induced as scheduled assuming the cluster was able to recover.
 
-Default implementation of the Healthcheck component allows to plug in any executable with a logic. A sample healthcheck would like like this:
+The default implementation of the Healthcheck component allows for plugging in any executable with logic. A sample healthcheck would like like this:
```
Comment (MayureshGharat) [inline docs/design.md]: like like written 2 times.

Comment (MayureshGharat) [inline docs/design.md]: Can you update this so that we can merge the request?

**Comment by gabrielhartmann** [comment]
@MayureshGharat: Apologies for the delay.  Was sidetracked with other work.  Should be good to merge now.

File: docs/design.md [hunk 2]
```diff
@@ -79,25 +79,25 @@ Rackspace|Rackspace API|not supported|TBD|
...
+Healthcheck is an optional component that allows you to control the damage inflicted against your service. If enabled, Atropos kicks off the healthcheck logic defined in the Fate Book before inducing a failure. The Healthcheck component needs to return success in order for the failure run. Otherwise the Scheduler skips the current failure cycle. This ensures that we are not aggravating any existing issues and lets the cluster fire self-healing routines and recover. If a healthcheck is not defined, failures will be induced as scheduled assuming the cluster was able to recover.
 
-Default implementation of the Healthcheck component allows to plug in any executable with a logic. A sample healthcheck would like like this:
+The default implementation of the Healthcheck component allows for plugging in any executable with logic. A sample healthcheck would like this:
```
Comment (MayureshGharat) [inline docs/design.md]: Ah, My bad should have caught this before. Thanks for the patch. Just a small comment. The line can be changed to either of this :

A sample healthcheck would be like this :
OR
A sample healthcheck would look like this :

**Comment by gabrielhartmann** [comment]
@MayureshGharat: Fixed missing word.

**Pull request merged**